# Logfile created on Mon May 18 23:17:41 -0700 2009 by /
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::EEXIST: File exists - /Users/leo/workspace/checkout/jgc/index>
#<Errno::EEXIST: File exists - /Users/leo/workspace/checkout/jgc>
#<Errno::EEXIST: File exists - /Users/leo/workspace/checkout>
#<Errno::EEXIST: File exists - /Users/leo/workspace>
#<Errno::EEXIST: File exists - /Users/leo>
#<Errno::EEXIST: File exists - /Users>
#<Errno::ENOENT: No such file or directory - >
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/index/production/activity>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
Mon May 18 23:20:15 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `loop'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<Mysql::Error: Table 'demo_production.activities' doesn't exist>
#<ActiveRecord::StatementInvalid: Mysql::Error: Table 'demo_production.activities' doesn't exist: SHOW FIELDS FROM `activities`>
#<MissingSourceFile: no such file to load -- mysql>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/config/ferret_server.yml>
Mon May 18 23:30:06 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `loop'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
Mon May 18 23:31:35 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `loop'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
Mon May 18 23:32:38 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Emails>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
#<MissingSourceFile: no such file to load -- mysql>
#<NameError: uninitialized constant Emails>
Mon May 18 23:35:48 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `loop'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
Mon May 18 23:37:46 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<fatal: exception reentered>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
Mon May 18 23:39:06 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:57:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `loop'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:44:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
Mon May 18 23:41:00 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
Mon May 18 23:41:59 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::ENOENT: No such file or directory - /Users/leo/workspace/checkout/jgc/log/mailer.rb.pid>
Tue May 19 00:13:21 -0700 2009
SIGTERM
["/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `sleep'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:42:in `watch'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:84:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `fork'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:64:in `start_with_pidfile'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/monitor.rb:111:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application_group.rb:113:in `create_monitor'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/application.rb:244:in `start'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/controller.rb:69:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:139:in `run'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `call'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons/cmdline.rb:105:in `catch_exceptions'", "/usr/local/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib/daemons.rb:138:in `run'", "./lib/daemons/mailer_ctl:25"]
